lib/Twig/Node/Module.php

```diff
@@ -384,10 +384,22 @@ protected function compileIsTraitable(Twig_Compiler $compiler)
 
     protected function compileDebugInfo(Twig_Compiler $compiler)
     {
+        $debugInfo = array_reverse($compiler->getDebugInfo(), true);
+        $env = $compiler->getEnvironment();
+
+        switch (false) {
+            case $env->isDebug():
+            case ($loader = $env->getLoader()) instanceof Twig_Loader_Filesystem:
+            case file_exists($file = $loader->getCacheKey($this->getAttribute('filename'))):
```

Comment:
This code defining a variable in a case statement to use it in the next one (but only when the condition is false) is totally unreadable.